README.md

```diff
@@ -36,3 +63,73 @@ The specification of these libraries can be found in [SEP](https://s1seven.githu
 ## License
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fs1seven%2Fschema-tools.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fs1seven%2Fschema-tools?ref=badge_large)
+
+## Contributing
+
+When updating the schema (links below), the tools need to be updated at the same time.
+
+Schema links:
+[EN10168](https://github.com/thematerials-network/EN10168-schemas)
+[E-CoC](https://github.com/thematerials-network/E-CoC-schemas)
+[CoA](https://github.com/thematerials-network/CoA-schemas)
+[CDN](https://github.com/thematerials-network/CDN-schemas)
+
+Process:
+
+1. Update the schema in question
+2. Check and update the types using the `create-schema-interfaces` CLI tool (paste in the path to the updated schema when prompted)
+3. If necessary, make changes in `generateContent.ts` and run `npm run build`.
+   If `generateContent.ts` has been changed and built, copy the contents of `dist/generateContent.js` to the schema repository and replace the contents of `generate-pdf.min.js` with the new minified code
+4. For a new Release Candidate, add a new fixture (used for testing)
+
+// Insert here instructions on how to create a new fixture
+
+## Starting out
+
+To get started, run `npm install`, `npm run bootstrap` and `npm run build`
+
+## Testing
+
+The following packages only test the latest version:
+
+- generate-coa-pdf-template
+- generate-en10168-pdf-template
+
+The remaining packages should be made compatible with [all schema versions][#supported schemas]
+
+### Fixtures
+
+- Create or use a folder following the schema name
+- Create a subfolder following the version of the release candidate
+- Add a valid json certificate with the name `valid_cert.json`
+- Add an invalid json certificate with the name `invalid_cert.json`
+- Add the `translations.json` - for ease of use we keep English and German translations in a single file
+- `certificate.ts`, `template_hbs.html` and `valid_cert.pdf` are dynamically generated with the following scripts:
+
+To generate `certificate.ts`, run
+
+```sh
+npm run fixtures:interfaces -- -s /Users/eamon/work/CoA-schemas/schema.json -o fixtures/CoA/v0.1.1/certificate.ts
+```
+
+from the root directory where -s indicates the path to the updated schema and -o indicates the path to the certificate to be generated.
+
+To generate `template_hbs.html`, run
+
+```sh
+npm run fixtures:html -- -c fixtures/CoA/v0.1.1/valid_cert.json -o fixtures/CoA/v0.1.1/template_hbs.html -t fixtures/CoA/v0.1.1/translations.json -T /Users/eamon/work/CoA-schemas/template.hbs
+```
+
+from the root directory.
+
+To generate `valid_cert.pdf`, run
```

Comment:
Same as above like To generate `valid_cert.pdf` (PDF certificate generated from JS generator)